tests/integration/framework/process/scheduler/scheduler.go

```diff
@@ -215,6 +219,17 @@ func (s *Scheduler) DataDir() string {
 	return s.dataDir
 }
 
+func (s *Scheduler) ETCDClient(t *testing.T) *client.EtcdClient {
+	t.Helper()
+
+	client := client.Etcd(t, clientv3.Config{
+		Endpoints:   []string{"127.0.0.1:" + s.EtcdClientPort()},
+		DialTimeout: 40 * time.Second, //nolint:mnd
```

Comment:
Linter is being removed

```suggestion
		DialTimeout: 40 * time.Second,
```

Reply:
this fixes a linter issue. anywhere there is a number it throws a lint err

Reply:
`mnd` has definitely been disabled though so we shouldn't need this.